dist/main.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+const core = require("@actions/core");
+const plivo = require("plivo");
+async function run() {
+    const from = core.getInput("fromPhoneNumber");
+    const to = core.getInput("toPhoneNumber");
+    const message = core.getInput("message");
+    core.debug(from, to, message);
+    const auth_id = core.getInput("PLIVO_AUTH_ID") || process.env.PLIVO_AUTH_ID;
+    const auth_token = core.getInput("PLIVO_AUTH_TOKEN") || process.env.PLIVO_AUTH_TOKEN;
+    var client = new plivo.Client(auth_id, auth_token);
+    var response = client.messages.create(from, to, message).then(function (response) {
+        console.log(response);
+    }, function (error) {
```

Comment:
I think, you would also need to throw the error, so that is being caught in the async execute (https://github.com/plivo/actions-sms/pull/1/files#diff-6e39535204d5f86b6aac471592c25c408c0b0d4222f4c77efa25b6626b7308caR23) and marked as failed

Reply:
Done